src/tests/Navbar.test.js

```diff
@@ -31,6 +32,30 @@ describe('Navbar Component', () => {
     expect(wrapper.find('LogoutAction')).toBeTruthy();
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should toggle the dropdown', () => {
+    const props = {
+      loggedIn: true,
+      history: { push: jest.fn() },
+      logoutAction: jest.fn(),
+    };
+    wrapper = shallow(<Navbar.WrappedComponent store={store} {...props} />);
+    const instance = wrapper.instance();
+    expect(wrapper.state().isOpen).toBe(false);
+    instance.toggle();
+    expect(wrapper.state().isOpen).toBe(true);
+  });
+  it('button click should toggle dropdown', () => {
+    const props = {
```

Comment:
Hey @AGMETEOR, seems you are repeating yourself here. The `props` and `shallow rendering`.  It'd be better to define those in a `beforeEach fn` inside the describe block so that its scope is limited to that describe block.

Reply:
@dondrzzy it's a good suggestion. However, I would be required to move the previously implemented beforeEach block within the describe block. This I feel is better suited for another ticket that refactors the tests in this file.